presto-client/src/test/java/io/prestosql/client/TestClientTypeSignature.java

```diff
@@ -60,6 +60,17 @@ public void testBackwardsCompatible()
         assertEquals(legacy, signature);
     }
 
+    @Test
+    public void testStringSerialization()
+    {
+        ClientTypeSignature bigint = new ClientTypeSignature(StandardTypes.BIGINT);
+        assertEquals(bigint.toString(), "bigint");
+        ClientTypeSignature varchar = new ClientTypeSignature(StandardTypes.VARCHAR, ImmutableList.of(ClientTypeSignatureParameter.ofLong(50)));
+        assertEquals(varchar.toString(), "varchar(50)");
+        ClientTypeSignature row = new ClientTypeSignature(StandardTypes.ARRAY, ImmutableList.of(ClientTypeSignatureParameter.ofType(new ClientTypeSignature(StandardTypes.BIGINT))));
```

Comment:
The variable represents an array type

```suggestion
        ClientTypeSignature array = new ClientTypeSignature(StandardTypes.ARRAY, ImmutableList.of(ClientTypeSignatureParameter.ofType(new ClientTypeSignature(StandardTypes.BIGINT))));
```

Anyway, since row has special handling path, it deserves a test cases too.

Reply:
Handled it. Thanks for pointing it out.